code/modules/mailorder/modr_entries.dm

```diff
@@ -0,0 +1,365 @@
+//information for catalogue sale entries
+ABSTRACT_TYPE(/datum/mail_order)
+/datum/mail_order
+	var/name = "Juicer Schweet's Spaghetti Western"
+	var/cleanname = null //sanitary version of name created for references
+	var/desc = "Be like the three-second elephant with heated value in space-bark."
+	var/list/order_items = list() // should not exceed 7 items, as mail order is sent in a box item
+	var/cost = PAY_UNTRAINED
+	var/list/order_perm = list() // optional access requirement to order a given item
+
+	New()
+		..()
+		src.cleanname = ckey(src.name)
+
+//Tanhony & Sons: journalistic equipment, primarily AV
+ABSTRACT_TYPE(/datum/mail_order/audiovideo)
+/datum/mail_order/audiovideo
+
+	audiolog
+		name = "Audio Log"
+		desc = "A must-have for catching that crucial conversation."
+		order_items = list(/obj/item/device/audio_log)
+		cost = PAY_TRADESMAN
+
+	audiotape
+		name = "Audio Tape"
+		desc = "Audio storage tape for use in suitable audio devices."
+		order_items = list(/obj/item/audio_tape)
+		cost = PAY_TRADESMAN / 3
+
+	laptop
+		name = "Personal Laptop"
+		desc = "The latest in micro-computing, right at your fingertips."
+		order_items = list(/obj/item/luggable_computer/personal)
+		cost = PAY_EXECUTIVE * 2
```

Comment:
While expensive, this does make an item that is currently limited by there being only one of them now accessible to anyone who can wait a few minutes for their wages to accumulate (or do a method of grinding for credits). Worth discussion.

Reply:
These are actually available from a seller in the debris field, hiding places on certain maps, court room lockers, the therapist spawns with one, and more than one azone!